src/components/Form/Input.js

```diff
@@ -29,6 +29,7 @@ const FormInput = ({
         />
         <input
           {...rest}
+          autoComplete="on"
```

Comment:
why? maybe you don't want autocomplete on some inputs, and how it currently is there is no way to override it

Reply:
I got hyped fixing console warnings. I'll move it back how it was then